app/main.py

```diff
@@ -1,8 +1,40 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[name] = self
+
+    def set_spouse(self, spouse_name: str) -> str:
```

Comment:
The `set_spouse` method is expected to return a string, but it currently does not return anything when a spouse is successfully set. Consider returning a confirmation message or status.